### ISSUE TITLE
Deprecate the classic notebook extension

### ISSUE THREAD

**Issue by jtpio** [voila-dashboards/voila-gridstack#71]
This will be a follow-up issue to https://github.com/voila-dashboards/voila-gridstack/pull/69

The idea is to reuse the same components as the ones used for the JupyterLab Extension, by having a standalone lab-based app that shoes the notebook and the editor only.

This is done in #69 in https://github.com/voila-dashboards/voila-gridstack/tree/0d5c9e3504c3d48a3aed004ea64528b7ea905fea/packages/gridstack-editor, and looks like the following:

![standalone-editor](https://user-images.githubusercontent.com/591645/100720762-c3665b00-33be-11eb-98d7-bea8fd3fda8a.gif)

The remaining items would be to:

- Bundle the standalone app in the distribution, for example under `voila-gridstack/editor` or `voila-gridstack/static` directly
- Add a button to the classic notebook UI to open the standalone app in a new browser tab
- Remove the classic notebook extension

**Comment by jtpio** [comment]
Thinking we might want to start deprecating the classic notebook extension as part of the `0.3.x` series.

Not sure whether we should provide the standalone editor as mentioned above. Users could just use JupyterLab instead.

Another option would be to see whether we want to make the current JupyterLab extension compatible with RetroLab.

**Comment by hbcarlos** [comment]
I was thinking the same. I would like to make this extension compatible with retrolab instead of using the standalone app.

We need to have two documents open to drag and drop cells from the notebook to the grid. Maybe we could create a map view of the notebook on a side panel?

**Comment by jtpio** [comment]
Yes. Probably RetroLab will start exposing the `left` and `right` areas at some point: https://github.com/jupyterlab/retrolab/issues/268

These areas will be hidden by default. With menu entries to show them on demand.

**Comment by hbcarlos** [comment]
We can start adding the widget to JupyterLab (sometimes it is helpful to open the grid editor without the notebook), and later we can easily create the extension for RetroLab.

> These areas will be hidden by default. With menu entries to show them on demand.

I suppose will be a command to open them, so, we can always execute the command when opening the grid editor.

**Comment by jtpio** [comment]
> We can start adding the widget to JupyterLab (sometimes it is helpful to open the grid editor without the notebook),

By "widget" this means the notebook mini-map? If so that sounds good, and it could go either to the left or right area.

> I suppose will be a command to open them

Most likely yes.
